CONTRIBUTING.md

```diff
@@ -29,19 +25,73 @@ for signing our CLA](https://www.citusdata.com/community/CLA)
    brew link openssl --force
    ```
 
-3. Get the code
+3. Get, build and test the code
```

Comment:
I'm an Oxford comma kind of guy. What's our house style? The `README` appears to use it more often than not. Can we add it here?